vim25/types/types.go

```diff
@@ -22115,6 +22116,34 @@ func init() {
 	t["HostConnectInfoNetworkInfo"] = reflect.TypeOf((*HostConnectInfoNetworkInfo)(nil)).Elem()
 }
 
+type ConnectPnicLLDPEndpointType struct {
```

Comment:
Thanks for the PR @vjadhav-arista . Haven't heard back from you on: https://github.com/vmware/govmomi/pull/3061#pullrequestreview-1346566916

Same problem here, vim25/{types,methods} are generated so this code will be overwritten next time we update. Further, the govc commands that use this method would only work against vcsim and not real vCenter.